Frontend/src/test/components/Nav/Nav.test.jsx

```
@@ -0,0 +1,129 @@
+vi.mock('firebase/auth', () => ({
+  signOut: vi.fn(),
+}));
+// Stub DesktopNavigation and MobileNavigation
+vi.mock('../../../components/Nav/DesktopNavigation', () => ({
+  __esModule: true,
+  default: props => (
+    <div data-testid="desktop-nav"
+         data-user={props.user ? 'true' : 'false'}
+         data-role={props.userData?.role || ''}
+         data-loading={props.loading ? 'true' : 'false'}>
+      <button data-testid="logout-btn" onClick={props.handleLogout}>Log out</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)